src/utils/registerDiscourseDatalogTranslators.ts

```diff
@@ -394,14 +394,16 @@ const registerDiscourseDatalogTranslators = () => {
                   dr.source === source ||
                   nodeLabelByType[dr.source] === source)
             );
-          return pageNames.filter((p) =>
+          pageNames = pageNames.filter((p) =>
             sourcedRelations.some((sr) =>
               matchDiscourseNode({
                 ...nodeByType[sr.target],
                 title: p,
               })
             )
           );
+          pageNames.push(...Object.values(nodeLabelByType).filter((p) => sourcedRelations.some((sr) => nodeByType[sr.target].text === p)));
+          return pageNames
```

Comment:
you can use `concat` instead of `let`:

```typescript
return pageNames.filter((p) =>
    sourcedRelations.some((sr) =>
        matchDiscourseNode({
                ...nodeByType[sr.target],
                title: p,
        })
    )
).concat(
    ...Object.values(nodeLabelByType).filter(
        (p) => sourcedRelations.some(
            (sr) => nodeByType[sr.target].text === p
        )
    )
)
```